### PR TITLE
Add completions for iftop

### PR DESCRIPTION
**Pull request by Xiretza**

### CODE REVIEW

File: share/completions/iftop.fish [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+complete -c iftop -s i -xa "(__fish_print_interfaces)" -d "Listen to packets on interface"
+complete -c iftop -s f -x -d "Use filter code to select the packets to count"
+complete -c iftop -s F -x -d "Filter to only specified IPv4 network"
+complete -c iftop -s G -x -d "Filter to only specified IPv6 network"
+complete -c iftop -s c -r -d "Use specified config file"
+complete -c iftop -s o -xa "(__fish_iftop_sort_orders)" -d "Sort by specified column"
+
+complete -c iftop -s t -d "Use text interface without ncurses"
+complete -c iftop -s L -x -d "Number of lines to print with -t"
+complete -c iftop -s s -x -d "With -t, print single output after specifiec number of seconds"
```
Comment (mqudsi) [inline share/completions/iftop.fish:29]: specific

File: share/completions/iftop.fish [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+complete -c iftop -s h -d "Print a summary of usage"
+complete -c iftop -s n -d "Don't do hostname lookups"
+complete -c iftop -s N -d "Don't resolve port number to service names"
+complete -c iftop -s p -d "Run in promiscuous mode"
+complete -c iftop -s P -d "Turn on port display"
+complete -c iftop -s l -d "Include link-local IPv6 addresses"
+complete -c iftop -s b -d "Don't display bar graphs of traffic"
+complete -c iftop -s m -x -d "Set the upper limit for the bandwidth scale"
+complete -c iftop -s B -d "Display bandwidth rates in bytes/sec rather than bits/sec"
```
Comment (mqudsi) [inline share/completions/iftop.fish:19]: I think this could be shortened with Show instead of Display and bytes/s + bits/s instead of /sec

**Comment by mqudsi** [comment]
LGTM besides the minor issues commented.

File: share/completions/iftop.fish (hunk 2 above)
Comment (EmilyGraceSeville7cf) [inline share/completions/iftop.fish:19]: As an idea for enhancing completion for this option: provide 'K', 'M' or 'G' suffixes when there is some number entered already as the current token. As for the code, the completion improvement can be based on [mine](https://github.com/fish-shell/fish-shell/pull/10757/files#diff-25723ae18a81f6705d4abcdf09095a75f106e6b05fc827fbff5d99c7a20a2a2fR3) can be used:

```fish
# Usage: __fish_complete_extensions extension1 extension2 ...
# Example: __fish_complete_extensions png jpg
function __fish_complete_extensions \
    --description 'Suggest provided extensions as $argv if the current token ends with a dot'

    set token (commandline -c -t)

    switch "$token"
        case '*.'
            for extension in $argv
                if string match --quiet --regex '\\\t|\t' -- $extension
                    echo -e "$token$extension"
                else
                    printf "%s%s\n" $token $extension
                end
            end
    end
end
```

**Review by EmilyGraceSeville7cf — COMMENTED** [review]
Please tell me whether my comments were useful or not. 😄

**Comment by zanchey** [comment]
Merged with rebase and review comments addressed as 421e27b1b

**Comment by Xiretza** [comment]
Oops, forgot about this, thanks.

**Pull request closed without merging**